addons/mail/static/src/models/message/message.js

```diff
@@ -456,6 +462,17 @@ function factory(dependencies) {
             return inlineBody.toLowerCase() === this.subtype_description.toLowerCase();
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsCurrentPartnerMentioned() {
+            if (!this.messaging.currentPartner) {
+                return false;
+            }
```

Comment:
I don't think we need it, `.includes(undefined)` will return `false` unless it is explicitly in the array (`[undefined]`) which should never be the case in our fields.